minifi-bootstrap/src/main/java/org/apache/nifi/minifi/bootstrap/RunMiNiFi.java

```diff
@@ -1703,7 +1760,10 @@ private void restartInstance() throws IOException {
 
     private static ByteBuffer performTransformation(InputStream configIs, String configDestinationPath) throws ConfigurationChangeException, IOException {
         try (ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream();
-                TeeInputStream teeInputStream = new TeeInputStream(configIs, byteArrayOutputStream)) {
+             TeeInputStream teeInputStream = new TeeInputStream(configIs, byteArrayOutputStream)) {
+
+            final Map<String, String> bootstrapSecurityProperites = new HashMap<>();
```

Comment:
A terrible nitpick on spelling of `Properties` that can be ignored. :)

Actually, `bootstrapSecurityProperites` doesn't look like it's used anywhere.

Reply:
yeesh.  Bootstrap Steak Frites. Let me clean that up.